### ISSUE TITLE
[ERROR] [parsers.hang]: Parsing error. Is your own_name ****** correct?

### ISSUE THREAD

**Issue by KingOfCoders** [MasterScrat/Chatistics#55]
I have done the command !python parse.py hangouts --own-name "**********", and then I get the above error. How can I fix this?

**Comment by MasterScrat** [comment]
You shouldn’t need to specify the own-name flag if you are importing more than one conversation. 

If you do use it, make sure the name you specificy is exactly the same as the one you use on the chat platform, including case, accents, etc.

**Comment by KingOfCoders** [comment]
When I don't specify the own-name flag, it gets my name correctly, but then at the end I still get that error.

**Comment by MasterScrat** [comment]
Ok, then it gets tricky to debug without seeing your JSON file. 

I added more debug output to this specific failure case, can you update the code and try again?

**Comment by KingOfCoders** [comment]
`2020-02-05 15:30:20,655 [INFO ] [parsers.hang]: Parsing Google Hangouts data...

2020-02-05 15:30:20,655 [INFO ] [parsers.hang]: Reading archive file raw_data/hangouts/Hangouts.json...

2020-02-05 15:30:21,047 [INFO ] [parsers.hang]: Trying to infer own_name from data...

2020-02-05 15:30:21,047 [INFO ] [parsers.hang]: Successfully inferred own-name to be First1 Last1
2020-02-05 15:30:21,047 [INFO ] [parsers.hang]: Extracting messages...
2020-02-05 15:30:21,051 [INFO ] [parsers.hang]: Assuming first1@randomorganization.org is First1 Last1
2020-02-05 15:30:21,052 [INFO ] [parsers.hang]: Assuming First2 Last2 is Fir2 Las2
2020-02-05 15:30:21,052 [INFO ] [parsers.hang]: Assuming First2 Last2 is Fir2 Las2
2020-02-05 15:30:21,053 [INFO ] [parsers.hang]: Assuming first1@randomorganization.org is First1 Last1
2020-02-05 15:30:21,053 [INFO ] [parsers.hang]: Assuming First2 Last2 is Fir2 Las2
2020-02-05 15:30:21,054 [INFO ] [parsers.hang]: Assuming First2 Last2 is Fir2 Las2
2020-02-05 15:30:21,054 [INFO ] [parsers.hang]: Assuming first1@randomorganization.org is First1 Last1
2020-02-05 15:30:21,054 [INFO ] [parsers.hang]: Assuming First2 Last2 is Fir2 Las2
2020-02-05 15:30:21,056 [INFO ] [parsers.hang]: Assuming First2 Last2 is Fir2 Las2
2020-02-05 15:30:21,056 [ERROR] [parsers.hang]: Parsing error. Is your own_name First1 Last1 correct?
2020-02-05 15:30:21,056 [ERROR] [parsers.hang]: Problem: this message was sent by First3 Last3, who is not you (First1 Last1)
2020-02-05 15:30:21,056 [ERROR] [parsers.hang]: It was sent by id 100138845835906592753, who is not your current interlocutor either (115735074538093235957)
2020-02-05 15:30:21,056 [ERROR] [parsers.hang]: Participants are: [{'gaia_id': '115735074538093235957', 'chat_id': '115735074538093235957'}, {'gaia_id': '117156097994960548283', 'chat_id': '117156097994960548283'}]`

That is the log now... I'm pretty sure that it breaks down when it sees a group chat. Is there any way for me to filter those out? Or is there another solution?

**Comment by MasterScrat** [comment]
Group chats are already ignored.

Did you change your name on Google at some point? Is one of the "Assuming..." log line referring to your own name?

Also can you try this branch: https://github.com/MasterScrat/Chatistics/tree/hangouts-ids

It's not merged yet, but it may solve your problem.

**Comment by KingOfCoders** [comment]
Yeah, I did change my name on Google at some point. In the log I replaced my name with "First1 Last1"

**Comment by KingOfCoders** [comment]
Note: got exact same error on new branch

**Comment by MasterScrat** [comment]
Arg. Ok thanks for the feedback, will have to look into this further...

**Comment by KingOfCoders** [comment]
Any updates?
